releasetasks/templates/desktop/binary_transparency.yml.tmpl

```diff
@@ -0,0 +1,38 @@
+{% set buildername = "release-{}-{}_binary_transparency".format(branch, product) %}
+-
+    taskId: "{{ stableSlugId(buildername) }}"
+    requires:
+        - "{{ stableSlugId('release-foo-{}_chcksms'.format(product)) }}"
```

Comment:
Not sure when you want to turn this on but you might want to change this to `"release-{}-{}_chcksms".format(branch, product)` to chain it to the *real* checksums task, when rolling this out in production.